### PR TITLE
GSoC 2022: Update webpage and create blogpost

### PR DESCRIPTION
**Pull request by Echolon**
- Update webpage
- Create blogpost

@wurstsalat3000 Github somehow tells me this: 

> The 'xsf/xmpp.org' repository doesn't contain the 'images/logos/GSoC_2022_Logo.png' path in 'master'. 

But is is actually correct I think. Any issues you see?

### CODE REVIEW

**Comment by wurstsalat3000** [comment]
@Echolon yeah, you can ignore that warning, since this URL is correct for the deployed website, where `/static/images//logos/GSoC_2022_Logo.png` becomes `/images/logos/GSoC_2022_Logo.png`.

File: content/blog/2022-05-28-gsoc2022-welcome.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+accepted (again!) as a hosting organization for XMPP projects, we also can welcome two new contributors who will work on open-source software projects in the XMPP environment! We have updated our [designated web-page](https://xmpp.org/community/gsoc-2022/) for the Google Summer of Code 2022 accordingly.
+
+### The XMPP projects at Google Summer of Code 2022
+
+So, please welcome **Patiga** and **PawBud** as new contributors! It is really great that you chose XMPP for your coding adventure!
+
+- **Patiga** will work on more [flexible file transfers in Dino](https://summerofcode.withgoogle.com/programs/2022/projects/z9ixHTWZ). Mentors will be **fiaxh** and **Marvin W.** - many thanks to both of you!
+  - Resource-wise, messenger applications tend to be on the lightweight side of the spectrum. This drastically changes when file transfers are added to the equation. File transfers can introduce arbitrary more resource-usage, both on network and data storage aspects. To alleviate this issue, stateless file sharing empowers the user to make informed decisions on which files to load. Deliverables:
```
Comment (vanitasvitae) [inline content/blog/2022-05-28-gsoc2022-welcome.md:18]: "File transfers can arbitrarily increase resource-usage"

"on which files to *down*load."

File: content/blog/2022-05-28-gsoc2022-welcome.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - Unified handling of http and jingle (peer-to-peer) file transfers
```
Comment (vanitasvitae) [inline content/blog/2022-05-28-gsoc2022-welcome.md:19]: "of HTTP and Jingle (peer-to-peer) file transfers"

**Review by vanitasvitae — CHANGES_REQUESTED** [review]

File: content/blog/2022-05-28-gsoc2022-welcome.md (hunk 2 above)
Comment (Echolon) [reply content/blog/2022-05-28-gsoc2022-welcome.md:19]: Quality quality quality - thanks!

File: content/blog/2022-05-28-gsoc2022-welcome.md (hunk 1 above)
Comment (Echolon) [reply content/blog/2022-05-28-gsoc2022-welcome.md:18]: 4c8671fa6ab239881bd221f5f63006073242bb2a

File: content/blog/2022-05-28-gsoc2022-welcome.md (hunk 2 above)
Comment (Echolon) [reply content/blog/2022-05-28-gsoc2022-welcome.md:19]: 4c8671fa6ab239881bd221f5f63006073242bb2a

**Pull request merged**